Source/WebKit/UIProcess/API/Cocoa/WKWebExtension.h

```diff
@@ -82,7 +82,7 @@ WK_CLASS_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA)) WK
  @discussion The app extension bundle must contain a `manifest.json` file in its resources directory. If the manifest is invalid or missing,
  or the bundle is otherwise improperly configured, an error will be returned.
  */
-+ (void)extensionWithAppExtensionBundle:(NSBundle *)appExtensionBundle completionHandler:(void (^)(WKWebExtension * WK_NULLABLE_RESULT extension, NSError * _Nullable error))completionHandler WK_SWIFT_ASYNC_THROWS_ON_FALSE(1);
++ (void)extensionWithAppExtensionBundle:(NSBundle *)appExtensionBundle completionHandler:(void (^)(WKWebExtension * _Nullable extension, NSError * _Nullable error))completionHandler NS_REFINED_FOR_SWIFT;
```

Comment:
is it guaranteed that either `extension` or `error` will be non-nil?

Reply:
`extension` is `nil` when `error` is not `nil`.